daliuge-translator/dlg/dropmake/web/main.js

```diff
@@ -202,14 +202,16 @@ function updateDeployOptionsDropdown() {
 async function checkUrlStatus(url) {
     return new Promise((resolve) => {
         $.ajax({
-            url: url,
-            type: 'HEAD',
-            dataType: 'jsonp',
+            url: url + 'api',
```

Comment:
Hi @awicenec - is this how we got around the CORS issue? The code was originally loading the base URL (which resolved to `visualiseDIM`), so now we make sure it is targeting the `URL/api` callback? 

I ask because you have added `HEAD` to the `daliuge_aware` decorator, but removed the decorator from the `visualiseDIM` method, so presumably that URL is no longer getting loaded (at least by this Javascript request).